### DISCUSSION TITLE
TLS Registry support

### DISCUSSION THREAD

**Discussion by antonwiens** [quarkiverse/quarkus-cxf#1485] [General]
Hi,

with latest quarkus updates a new tls registry is provided by quarkus. Is the tls registry already supported by quarkus cxf or does it need implementation? If its not supported, is this a goal for the near future? I would really appreciate that.

Best regards
Anton Wiens

**Comment by ppalaga** [comment]
Thanks, @antonwiens, that would be nice to have. Would you mind filing a new issue?

**Reply by ppalaga** [reply]
The feature request is tracked as https://github.com/quarkiverse/quarkus-cxf/issues/1486
